allennlp/modules/transformer/t5.py

```diff
@@ -14,23 +13,23 @@
 
 from allennlp.common import FromParams, Params, Lazy, Registrable
 from allennlp.common.checks import ConfigurationError
-from allennlp.modules.transformer.transformer_module import (
-    TransformerModule,
+from allennlp.modules.transformer import TransformerModule
+
```

Comment:
There's an extra space here, and we usually use the most specific form of the import here when within the same parent module:

```suggestion
from allennlp.modules.transformer_module import TransformerModule
```

Reply:
This avoids a potential circular import issue, which could otherwise happen if anything in the `t5` module is imported in `allennlp/modules/transformer/__init__.py` before `TransformerModule`.